iib/workers/tasks/build_merge_index_image.py

```diff
@@ -344,3 +357,15 @@ def is_bundle_version_valid(bundle_path, valid_ocp_version):
         )
 
     return False
+
+def sort_floats(versions):
+    """
+    Sort list of strings with floats
+
+    :param list versions: unsorted list of versions
+    :rtype: list
+    """
+    convert = lambda text: int(text) if text.isdigit() else text
+    alphanum_key = lambda key: [convert(c) for c in re.split('([0-9]+)', key)]
+    versions.sort(key=alphanum_key)
```

Comment:
This sorts the given list in place. Either use `sorted` instead, or don't return the list.